### DISCUSSION TITLE
are devtools ready for 3?

### DISCUSSION THREAD

**Discussion by dusty** [TanStack/query#1210] [Q&A] [answered]
I'm getting an error trying to run devtools using the 3 beta.  My assumption is that it still needs to be updated to support v3, but wanted to check.  I didn't see anything in the migration notes on the beta doc page or in the devtools repo.

PS.  new to discussions, it seems this would be a better place than an issue for random questions, is that correct?

<img width="942" alt="Screen Shot 2020-10-23 at 10 59 12 " src="https://user-images.githubusercontent.com/792/97037904-d31c9500-151e-11eb-8fb7-a04dd9a66bb4.png">

**Comment by TkDodo** [comment, marked as answer]
I think in this case, we would need an issue in the react-query-devtools repo. I also see no issue or branch there, so I think it is just not yet done.

**Reply by TkDodo** [reply]
there has been a first release, see: https://github.com/tannerlinsley/react-query-devtools/issues/78#issuecomment-727088865
